### ISSUE TITLE
Update formula to install rnp.dylib (or add rnp-dev).

### ISSUE THREAD

**Issue by ni4** [riboseinc/homebrew-rnp#29]
It would be good to be able to install rnp.dylib together with rnp/rnpkeys utilities, which would be used by fellow Swift developers once bindings are available.

**Comment by ni4** [comment]
@ribose-jeffreylau should we use separate rnp-dev formula, or include some flags to the main one? Not sure which is more common for homebrew.

**Comment by ribose-jeffreylau** [comment]
@ni4 I'm actually not too aware of "-dev" formulas in homebrew, so I'm leaning towards adding flags (or whatever) to the main formula.

**Comment by ronaldtse** [comment]
@ni4 @ribose-jeffreylau why not just install rnp.dylib by default in the homebrew formula?

**Comment by ribose-jeffreylau** [comment]
> @ni4 @ribose-jeffreylau why not just install rnp.dylib by default in the homebrew formula?

That seems to make most sense to me.

**Comment by ni4** [comment]
Agree, installing dylib + dynamically linked CLI should be the same in sense of time and space as having statically-linked CLI, just one file more.
